scss/backend/dot-admin/portlets/_content-edit.scss

```diff
@@ -83,13 +83,8 @@ $sidebar-width: 220px;
 
 .content-edit__history-version .dijitContentPane {
   padding: 0px;
-
-  .contentIdentifier {
-    font-weight: bold;
-  }
 }
 
-.content-edit__divider {
-  background-color: $gray-ligther;
-  margin: 20px 0;
+.portlet-sidebar {
+  width: 220px;
```

Comment:
Why do we need this? I think we are hardcoding sidebar width because each portlet could be different.